src/meltano/core/logging/formatters.py

```diff
@@ -26,23 +26,55 @@ def _process_formatter(processor: Processor):
 
     Args:
         processor: A structlog message processor such as structlog.dev.ConsoleRenderer.
+
+    Returns:
+        A configured log processor.
     """
     return structlog.stdlib.ProcessorFormatter(
         processor=processor, foreign_pre_chain=LEVELED_TIMESTAMPED_PRE_CHAIN
     )
 
 
-def console_log_formatter(colors: bool = False) -> None:
-    """Create a logging formatter for console rendering that supports colorization."""
-    return _process_formatter(structlog.dev.ConsoleRenderer(colors=colors))
+def console_log_formatter(
+    colors: bool = False, exception_formatter: str = "plain"
+) -> None:
```

Comment:
I just noticed the annotated return type is wrong:

```suggestion
) -> structlog.stdlib.ProcessorFormatter:
```